webapp/wct.conf.json

```diff
@@ -11,12 +11,14 @@
           "no-sandbox"
         ],
         "firefox": [
-          "--headless"
```

Comment:
Did this flag actually change in Firefox 70?

Reply:
No. `firefox --help` has always shown the single-dash form, but takes the double-dash form implicitly (at least on Linux). I changed it to avoid relying on undocumented behaviours.